docs/hub/repositories.md

```diff
@@ -11,5 +11,8 @@ In these pages, you will go over the basics of getting started with Git and inte
 - [Pull Requests & Discussions](./repositories-pull-requests-discussions)
   - [Pull Requests advanced usage](./repositories-pull-requests-discussions#pull-requests-advanced-usage)
 - [Webhooks](./webhooks)
+- [Notifications](./notifications)
+- [Collections](./collections)
```

Comment:
I also took the opportunity to add those 2 pages to the list as they are already in the sidebar menu cc @julien-c @osanseviero

Reply:
Good idea!